nixos/roles/ai-model-server.nix

```diff
@@ -158,6 +158,35 @@ in
             taginclude = [
               "host"
               "name"
+              "gpu_id" # gpu index is still useful to distinguish multiple GPUs in the same system
```

Comment:
Why do you have `gpu_id` here? The `name` tag already gives us a way to distinguish multiple GPUs ("card0", *card1", etc). Checking against the data which the telegraf plugin pulls from rocm-smi on one of the production GPU servers, `gpu_id` seems to be the same for each GPU (0x744a). Unless your goal is to distinguish between different GPU models installed in the same chassis?

Reply:
I don't quite understand the reasoning behind not including the GPU UUID. The GPUs are high value low-quantity assets and I see values tracking their data even when moved across hosts